webui/react/.eslintrc.js

```diff
@@ -35,18 +36,19 @@ module.exports = {
       files: ['**/src/e2e/**'],
       rules: {
         '@typescript-eslint/no-floating-promises': 'error',
+        'no-console': 'off',
```

Comment:
this is the change i made

Reply:
do we wanna always ignore console logs? or just specific cases? 
if ignoring it in spefic cases, using the inline disable comment might be better?

Reply:
that's a good point. i wouldn't expect a logger to exist anywhere else